README.md

```diff
@@ -1,2 +1,127 @@
-# clever-csharp
-C# client library for Clever APIs
+# Clever - the C# library for the Clever API
+
+## Installation and API Documentation
+View more detailed documentation [here](docs/README.md)
+
+<a name="getting-started"></a>
+## Getting Started
+
+```csharp
+using System;
+using System.Diagnostics;
+using Clever.Api;
+using Clever.Client;
+using Clever.Model;
+
+namespace Test
+{
+    public static class Program
+    {
+        public static void Main()
+        {
+
+            // Configure OAuth2 access token for authorization: oauth
+            Configuration.Default.AccessToken = "DEMO_TOKEN";
```

Comment:
Or at minimum, include a comment around each of these that warns users appropriately around token best-practices

https://dev.clever.com/docs/security#section-security-best-practices